spec.bs

```diff
@@ -255,9 +267,12 @@ const publicKey = {
   authenticatorSelection: {
     userVerification: "required"
   },
+  
+  **TODO**: Extend definition (perhaps in future) to make this optional (so to only have userPresence: "required", and thereby support regions outside of europe) and remove friction of having to type in User PIN's?
 
-  timeout: 360000,  // 6 minutes
-
+  timeout: 60000,  // 1 minute
```

Comment:
No objection to the change to an example, I just stole this from WebAuthn examples anyway :P